### PR TITLE
Add links to Tox's Spectrum community

### PR DESCRIPTION
**Pull request by SkyzohKey**
I'm currently working very hard on Tox's rebranding, to make it more « sexy » and « appealing » and this is one of the steps required.

I tought it would be cool to have a sort of message board mixed with chat and found Spectrum that looks really interesting. I've always wanted to setup a forum/chat for Tox's community (@grayhatter too iirc) and finally made it!

This PR scope is adding badge + link to the community to make it more discoverable.
I'll open a PR on the toktok.ltd / tox.chat websites too.

IMHO, this doesn't duplicate IRC community and should allows non tech-savvy people to enter the great community that Tox is. The only thing required is to spread the wor(l)d about this new place to discuss Tox's stuff!

I hope you'll like this! :yum:

Oh and feel free to join us on Spectrum too! https://spectrum.chat/tox

---
This change is [Reviewable](https://reviewable.io/reviews/toktok/c-toxcore/829)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
 # ![Project Tox](https://raw.github.com/TokTok/toxcore/master/other/tox.png "Project Tox")
 
-**Current build status:** [![Build Status](https://travis-ci.org/TokTok/c-toxcore.svg?branch=master)](https://travis-ci.org/TokTok/c-toxcore)
-**Current Coverage:** [![Coverage Status](https://coveralls.io/repos/github/TokTok/toxcore/badge.svg?branch=master)](https://coveralls.io/github/TokTok/toxcore?branch=master)
+[![Build Status](https://travis-ci.org/TokTok/c-toxcore.svg?branch=master)](https://travis-ci.org/TokTok/c-toxcore) [![Coverage Status](https://coveralls.io/repos/github/TokTok/toxcore/badge.svg?branch=master)](https://coveralls.io/github/TokTok/toxcore?branch=master) [![Join the community on Spectrum](https://withspectrum.github.io/badge/badge.svg)](https://spectrum.chat/tox)
```
Comment (iphydf) [inline README.md]: Are there "IRC" badges?

Comment (SkyzohKey) [reply README.md]: @iphydf Added! :)

**Comment by GrayHatter** [comment]
Our Messenger/Communication client needs a 3rd separate communication system...

Right... I agree with @nurupo's reaction...

**Comment by SkyzohKey** [comment]
@GrayHatter In the current state of Tox, I feel that letting ''average user'' a way to contact us without Tox/IRC is a great choice. Also, it permits to have conversations history. Feel free to join us on Spectrum and try, then you can understand why this is a great alternative to IRC. You get everything from a message board (original thread, persistence, formating) plus the convenience of an IRC-like chat that is also persisted ;)

**Comment by tox-user** [comment]
I don't see anything wrong with spreading the word about Tox on other platforms. What do we have to lose?

**Comment by iphydf** [comment]
@tox-user: the risks are community fragmentation and confusion. Devs are unlikely to be in the spectrum community, but putting a link to it on the repo may lead people to think that they are.

**Comment by tox-user** [comment]
Good point. Maybe we should add it to tox.chat then? It already has links to facebook and twitter (surprisingly no reddit though) and I don't think anyone is active on those websites. I think that things like this might help to make Tox a little more popular.

**Comment by SkyzohKey** [comment]
@iphydf No risk, no gain. And you're free to join us at Spectrum! :)

**Comment by SkyzohKey** [comment]
@iphydf No risk, no gain. And you're free to join us at Spectrum! :)

**Comment by zoff99** [comment]
@iphydf @SkyzohKey i just thinks its very strange that all the devs of some communication software, are NOT using their own software.
and now a new (non Tox) way of communication is proposed?

the best thing would be to work collectively to fix Tox, so that devs can do all their group communication via Tox. otherwise just give up and stop now

**Comment by SkyzohKey** [comment]
@zoff99 
It would be nice to use Tox to communicate with the community (ie. the Tox groupchat) but in the reality, Tox isn't a social IM yet.
Of course I'd like to see the Tox groupchat full of new people everyday, but we've seen it dying day after day. Currently we moved to the Tox group on Wire, until Tox groups are feature-complete. But this as nothing to do with groupchats, just with online presence.

We need the "Tox" word to become a verb, just as Skype did. ie. « Hey Dave, wanna Tox tonight? ». And I have conviction that acheiving this can only be possible by spreading the world, which is what i'm doing currently. Plain marketing.

I'm not really sure that Tox is designed to be a message board / forum software. We are making an Instant Messaging app, not a forum one. People are already on IRC. Spectrum is in the middle, Forum-like thread with real-time chat replies. This just allows non tech-savvy guys to join and ask questions about everything Tox related without having to install it.

**That's a marketing principle.**

You need to build communities around your software; IRC is for people that understand IT, Spectrum is for people that just use their computers to Facebook, YouTube, Mail & Netflix. Remember, those are the people we target with Tox. We aim to provide an easy to use, zero config IM app for the average user to have the choice to chat securely with people that matters.

Now if you prefer to only target people that actually cares about their privacy, to have privacy via Tox, that's fine too. But then we need to re-target how Tox is built.

I'm conviced that this will help bringing Tox to new users. What marketers call "aquisition". We need that. And it can only be accomplished by spreading the world, maintaining a strong brand image and in the mean time, developing better IM solutions.

The project's philosophy has been lost when the hard-fork hapened, I'm trying to restore it, without being a dumbass telling everybody that we are wrong. We are not wrong, we just need to understand how **real users** discover & uses our apps. ;)

(this post is **not** meant to be offensive, it only reflects my own opinion about developing a brand image that rocks! :yum:)

**Comment by tox-user** [comment]
@zoff99 I agree that more developers should use Tox, but I also agree with @SkyzohKey that we need marketing.

>The project's philosophy has been lost when the hard-fork hapened

What do you mean by that?

**Comment by SkyzohKey** [comment]
@zoff99 @tox-user the Spectrum community I've setup is meant principaly for users. Devs can participate too of course.

When I say that the philosophy has been lost, that's because I have the feeling that Tox is now developed for developers, not users. And our audience is not developers. The Tox audience always had been peoples. The average one. Comfortable enough to install Tox, but not an IT guy. Tox branding currently gives the impression that the software is for everyone but average user. (ie. The website scheme/icons etc).

I'm trying to put effort in making Tox more discoverable & user friendly. You can see such an attempt with the icons I've designed last night ->

Tox new illustrations style

**Comment by iphydf** [comment]
@zoff99, @sudden6, @nurupo: do you think this PR should be closed?

**Comment by nurupo** [comment]
Yeah.

**Comment by iphydf** [comment]
Ok, let's revisit this work later.

**Pull request closed without merging**